run_dir/design/ont_flowcell.html

```diff
@@ -0,0 +1,153 @@
+{% extends 'base.html' %}
+{% block stuff %}
+{% import re %}
+<div id="page_content">
+    <h1>ONT run <span id="page_title">{{ flowcell.get('name') }}</span></h1>
+
+    <div class="row">
+
+      <div class="col-md-4 borderRight" style="width: 500px; margin-right: 120px">
+        <h4 style="margin-top: 20px; margin-bottom: 10px;">Project</h4>
```

Comment:
Use bootstrap classes to set margin and padding.

Reply:
d6eafddef38dd61c0d94629344c647767212d3ad